kotlin.kt

```diff
@@ -1,20 +1,16 @@
-public fun main(args: Array<String>) {
-	for (i in 1..100) {
-		fizzBuzz(i)
-	}
+inline fun Int.divisibleBy(x: Int) = this % x == 0
+
+fun main(args: Array<String>) {
+	(1..100).forEach(::fizzBuzz)
```

Comment:
And this `::function` notation is also something I'm unfamiliar with, is it basically... some kind of bind? so you're mapping all those values into it?